src/main/telemetry/msp_shared.c

```diff
@@ -86,12 +86,12 @@ void sendMspErrorResponse(uint8_t error, int16_t cmd)
     sbufSwitchToReader(&mspPackage.responsePacket->buf, mspPackage.responseBuffer);
 }
 
-bool handleMspFrame(uint8_t *frameStart, uint8_t *frameEnd)
+bool handleMspFrame(uint8_t *frameStart, uint8_t *frameEnd, bool force)
```

Comment:
boolean parameters make code difficult to read and maintain.
If you see ``` handleMspFrame(frameStart, frameEnd, true);``` you have no idea what `true` means.

Instead create an `enum` with a descriptive name for each of the two values.

Reply:
Fair enough.